packages/phone-number-privacy/signer/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@celo/phone-number-privacy-signer",
-  "version": "1.1.6-dev",
+  "version": "1.1.8-dev",
```

Comment:
Note: Version 1.1.7 had monitoring changes that we only needed for cLabs signers, so it was not released to 3rd party signers. We also forgot to update this package.json before we tagged the image :/